app/domain/ptime/assign_employee_ids.rb

```diff
@@ -0,0 +1,69 @@
+# This script will assign each person the corresponding employee ID from PuzzleTime
+module Ptime
+  class AssignEmployeeIds
+
+    # rubocop:disable Rails/Output
+    def run(should_map: false)
+      puts 'Notice this is a dry run and mapping will not happen!' unless should_map
+      fetch_data
+
+      puts "Currently there are:
+      - #{@ptime_employees.length} employees in PuzzleTime
+      - #{@skills_people.count} people in PuzzleSkills
+      This is a difference of #{(@skills_people.count - @ptime_employees.length).abs} entries"
+      map_employees(should_map)
+    end
+
+    private
+
+    # rubocop:disable Metrics
+    def map_employees(should_map)
+      puts 'Assigning employee IDs now...' if should_map
+
+      ambiguous_entries = []
+      unmatched_entries = []
+      mapped_people_count = 0
+
+      @ptime_employees.each do |ptime_employee|
+        ptime_employee_firstname = ptime_employee['attributes']['firstname']
+        ptime_employee_lastname = ptime_employee['attributes']['lastname']
+        ptime_employee_name = "#{ptime_employee_firstname} #{ptime_employee_lastname}"
```

Comment:
Don't assign the variables on line 28 and 29 just do the hash digging here and save those 2 lines